CHANGES.md

```diff
@@ -16,6 +16,10 @@ wavesurfer.js changelog
   - Use of default `edgeScrollWidth` value no longer dependent on regions being created via
     plugin params (#2401)
 
+5.3.0 (21.11.2021)
+------------------
+- Regions plugin: disable `region-remove` event emission during plugin teardown (#2403)
```

Comment:
this change will be part of 6.0.0 (which is unreleased atm)

Reply:
Sorry about that. Updated. Let me know if you want me to squash these commits (you may be planning to do that, but wanted to mention it).